src/shared/telemetry/telemetryUtils.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as vscode from 'vscode'
+import { ext } from '../extensionGlobals'
+import { Datum, ResultWithTelemetry } from './telemetryEvent'
+
+export function defaultMetricDatum(name: string): Datum {
+    return {
+        name: name,
+        unit: 'Count',
+        value: 1
+    }
+}
+
+export function registerCommand<T>(
```

Comment:
We're trying to switch to single-parameter arguments in new code:

```
export function registerCommand<T>({
    command,
    callback,
    thisArg,
    register = vscode.commands.registerCommand
}: {
     command: string,
     callback: (...args: any[]) => (Promise<ResultWithTelemetry<T> | void>),
     thisArg?: any,
     register: typeof vscode.commands.registerCommand
})
```

or

```
export function registerCommand<T>({
    register = vscode.commands.registerCommand,
    ...args
}: {
     command: string,
     callback: (...args: any[]) => (Promise<ResultWithTelemetry<T> | void>),
     thisArg?: any,
     register: typeof vscode.commands.registerCommand
})
```

Reply:
:+1: